CHANGELOG.md

```diff
@@ -462,6 +482,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ### Changed
 
 - Upgraded all Cumulus workflow tasks to use `@cumulus/cumulus-message-adapter-js` version `2.0.1`
+- **CUMULUS-1823**
+  - Updates to Cumulus rule/provider schemas to improve field titles and descriptions.
```

Comment:
This Changelog note seems out of place, both here and on `master`

Reply:
Should this be fixed here, or fixed in master and merged?

Reply:
I can fix it in the audit fail PR and merge main back here, as this PR can't succeed until we get that sorted.

Reply:
That would have worked, but it looks like we didn't do that in the audit PR. I'm fine with just fixing it here, which will eventually make it upstream to master

Reply:
Indeed, I'd more or less decided to do that, apologies for not covering it here sooner.   Other issues to sort with CI first.

Reply:
Confirmed these were bad/committed after the release - bad merge/etc: 

https://github.com/nasa/cumulus/commit/6d00253efb2b4ad3287420e88a4a0447c5e2a20b#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

https://github.com/nasa/cumulus/commit/b613ba43c5c814910b566b8ce442f893fd0aaaaa

----

69077c726